components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -1024,13 +1024,17 @@ public static String getTileFilename(int tileX, int tileY,
    * @throws IOException Never actually thrown.
    */
   public static String getFilename(int series, int image, IFormatReader r,
-    String pattern) throws FormatException, IOException
+    String pattern, boolean padded) throws FormatException, IOException
```

Comment:
As this is breaking public API, can you also re-add the previous method signature calling this implementation with  `padded=false`?